examples/js/exporters/GLTFExporter.js

```diff
@@ -366,6 +347,45 @@ THREE.GLTFExporter.prototype = {
 
 		}
 
+		/**
+		 * Process and generate a BufferView from an image Blob.
+		 * @param {Blob} blob
+		 * @return {Promise<Integer>}
+		 */
+		function processBufferViewImage( blob ) {
+
+			if ( ! outputJSON.bufferViews ) {
+
+				outputJSON.bufferViews = [];
+
+			}
+
+			return new Promise( function (resolve) {
+
+				var reader = new window.FileReader();
+				reader.readAsArrayBuffer( blob );
+				reader.addEventListener('loadend', function () {
```

Comment:
Needs a space before `'loadend'`.

Reply:
And how about using `.onloadend` for the consistency with other parts in the exporter?

Reply:
Done ✅